vpn/muxer.go

```diff
@@ -93,7 +93,9 @@ type muxer struct {
 
 	// eventListener is a channel to which Event_*- will be sent if
 	// the channel is not nil.
-	eventListener chan uint16
+	eventListener chan uint8
+
+	failed bool
```

Comment:
what does this field do?

Reply:
this is an unrelated change, I will force-push (moved to #38)